Marlin/stepper.cpp

```diff
@@ -601,14 +639,28 @@ void Stepper::init() {
   TCNT1 = 0;
   ENABLE_STEPPER_DRIVER_INTERRUPT();
 
-  #if ENABLED(ADVANCE)
+  #if ENABLED(ADVANCE) || ENABLED(LIN_ADVANCE)
+
+    #if ENABLED(LIN_ADVANCE)
+
+      for (int i = 0; i < EXTRUDERS; i++) {
```

Comment:
Note that in the near future, a single extruder might have multiple steppers. So `e_steps` will not depend on `EXTRUDERS` but probably something named `E_MOTORS` or similar.